browser/about_flags.cc

```diff
@@ -377,24 +377,24 @@ const flags_ui::FeatureEntry::Choice kBraveSkusEnvChoices[] = {
     {"native-brave-wallet",                                                    \
      flag_descriptions::kNativeBraveWalletName,                                \
      flag_descriptions::kNativeBraveWalletDescription,                         \
-     kOsDesktop | flags_ui::kOsAndroid,                                        \
+     kOsDesktop | kOsAndroid,                                                  \
```

Comment:
Use kOsAndroid directly to avoid confusion.
See https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/about_flags.cc;l=295 in upstream